### ISSUE TITLE
Data split mentioned in the README

### ISSUE THREAD

**Issue by AngledLuffa** [UniversalDependencies/UD_Kazakh-KTB#3]
The README mentions an 80/10/10 data split.  However, the test set is 20x larger than the training set, and there is no dev set at all.  Is there a large reservoir of missing data, is the README incorrect, or is there a different recommended division of the available data?

**Comment by dan-zeman** [comment]
I think the README is incorrect (perhaps it reflects original plans that were never met). Kazakh is one of the languages that barely passed the 10k-test-word threshold for CoNLL shared tasks, and the “train” set was just a tiny sample for the participants to get a glimpse of what the data looks like.
